telematrix/__init__.py

```diff
@@ -229,14 +230,14 @@ def get_username(user_id):
                     continue
 
                 if content['msgtype'] == 'm.text':
-                    msg, mode = format_matrix_msg('<{}> {}', displayname, content)
-                    response = await group.send_text(msg, parse_mode=mode)
+                    msg, mode = format_matrix_msg('{}', content)
+                    response = await group.send_text("<b>{}:</b> {}".format(displayname, msg), parse_mode='HTML')
                 elif content['msgtype'] == 'm.notice':
-                    msg, mode = format_matrix_msg('[{}] {}', displayname, content)
-                    response = await group.send_text(msg, parse_mode=mode)
+                    msg, mode = format_matrix_msg('{}', content)
+                    response = await group.send_text("[{}] {}".format(displayname, msg), parse_mode=mode)
                 elif content['msgtype'] == 'm.emote':
-                    msg, mode = format_matrix_msg('* {} {}', displayname, content)
-                    response = await group.send_text(msg, parse_mode=mode)
+                    msg, mode = format_matrix_msg('{}', content)
+                    response = await group.send_text("* {} {}".format(displayname, msg), parse_mode=mode)
```

Comment:
Is there a reason why you don't use `format_matrix_msg` to do the formatting?

Reply:
I don't exactly remember why I did it this way, but I removed the username parameter from the format_matrix_msg function signature to add the username outside that function instead. The message always has to have HTML formatting now since the username is in bold.